tests/suites/1_stateful/08_select_stage/08_00_parquet/08_00_06_transform.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../../shell_env.sh
+
+echo "drop table if exists t1;" | $MYSQL_CLIENT_CONNECT
+echo "CREATE TABLE t1 (id INT, age INT);" | $MYSQL_CLIENT_CONNECT
+echo "insert into t1 (id, age) values(1,3), (4, 6);" | $MYSQL_CLIENT_CONNECT
+
+DATADIR_PATH="/tmp/08_00_00"
+rm -rf ${DATADIR_PATH}
+DATADIR="fs://$DATADIR_PATH/"
+echo "copy into '${DATADIR}' from t1 FILE_FORMAT = (type = PARQUET);" | $MYSQL_CLIENT_CONNECT
+
+echo "drop stage if exists s2;" | $MYSQL_CLIENT_CONNECT
+echo "create stage s2 url = '${DATADIR}' FILE_FORMAT = (type = PARQUET);"  | $MYSQL_CLIENT_CONNECT
+
+echo '--- copy 1'
+echo "copy into t1 from (select (t.id+1), age from @s2 t)  FILE_FORMAT = (type = parquet);" | $MYSQL_CLIENT_CONNECT
```

Comment:
We only support parquet format, how about add an error test for:
copy into t1 from (select (t.id+1), age from @s2 t)  FILE_FORMAT = (type = csv)?

Reply:
done